admin/run

```diff
@@ -84,8 +84,10 @@ pre_command='carton exec --'
 echo "Running '$command_verbatim $*' and logging at '$log_file_path'..."
 if [[ -t 1 ]]
 then
+  echo "Running $command_verbatim $*" | TZ=Z ts %FT%.TZ | tee -a "$log_file_path"
```

Comment:
For this case, won't it now be logged to standard output twice due to the `echo` above?

Reply:
Yes. The `echo` above was also informing of the log file path.

I added the commit https://github.com/metabrainz/musicbrainz-server/pull/3310/commits/04e1202db975b7d121f819e78b9ee06739c7a6c9 to avoid any duplication.